app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -97,7 +149,7 @@ class ProfileListAdapter(
     }
 
     fun removeWork(tag: String) {
```

Comment:
Remve this whole function. Directly place WorkManagerHelper.cancelWork(tag) in place of removeWork(tag).

Reply:
@rob729 I have replaced all the usages of this function, and hence removed it